src/js/jsx/PanelSet.jsx

```diff
@@ -269,11 +276,19 @@ define(function (require, exports, module) {
             this.getFlux().actions.preferences.setPreferences(nextState);
             headlights.logEvent("user-interface", "panel-toggle", panelName);
         },
+        
+        /**
+         * True if the current docuemnt is initialized and its panels should be displayed.
+         * 
+         * @return {Boolean}
```

Comment:
Type expressions for primitive types are always lower case: `boolean`, `number`, `object`, etc.